src/styles.css

```diff
@@ -2,15 +2,52 @@
   font-family: sans-serif;
   text-align: center;
 }
-.homeview-user-names    {
-    font-family: Arial, Helvetica, sans-serif;
-    text-transform: capitalize;
+
+.homeview-user-names {
+  font-family: Arial, Helvetica, sans-serif;
+  text-transform: capitalize;
 }
 
-a:link, a:visited {
+a:link,
+a:visited {
   color: black;
   padding: 14px 25px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
 }
+
+.profile-view__background {
+  background-color: lightgrey;
+  background: url(https://img.lemde.fr/2019/04/22/0/191/1619/1079/688/0/60/0/e39da8d_2FIads9h8wB-0SwSgxVaVWsp.jpg)
+    lightgrey top center no-repeat;
+  background-size: 300px, contain;
+  margin-top: 70px;
+  margin-left: 100px;
```

Comment:
This will create an issue 

![Screenshot 2020-03-05 at 10 42 50](https://user-images.githubusercontent.com/34084867/75968852-59bc2d80-5ece-11ea-909f-512e965e02c3.png)

Reply:
It only affects the background image of the container. Once we all decide which images we will be using for the background we can change it.

Reply:
We can change this once we've decided how we want the layout, and we make the App responsive.